djangogirls/settings.py

```diff
@@ -1,8 +1,14 @@
 import os
 import dj_database_url
+import dotenv
 import sentry_sdk
 from sentry_sdk.integrations.django import DjangoIntegration
 
+LOCAL_DEVELOPMENT = os.getenv("LOCAL_DEVELOPMENT", False)
+
+if LOCAL_DEVELOPMENT:
+    dotenv.read_dotenv(os.path.join(os.path.dirname(os.path.dirname(__file__)), '.env'))
```

Comment:
Why only local? Wouldn't it make sense to always use the `.env` file no matter what environment we're in?

Reply:
I'm unfamiliar with the deployment process for this service however I typically use `.env` files for local development, and rely on configured environment variables on the instance hosting the application to provide the required values. 

@ramonsaraiva, I'm happy to make the change if there is a preference to always use the `.env` file.

Reply:
Not sure if there is a preference.
I usually use `.env` everywhere. Simply inject the proper `.env` for the proper environment.

Reply:
I'll make the change this evening to use `.env` 👍

Reply:
Cool! Let's just make sure that we include the proper `.env` in the deployment process. I'm not sure who owns that but since we'll be relying on it, we need to make sure it is in the right place.

Reply:
Looking at the settings file, seems like we're already grabbing values from `os.environ` so we should be good.

Reply:
Thanks for the feedback @ramonsaraiva. It's been actioned 👍please let me know if there is anything else I can do to help this PR get accepted :)

Reply:
I looks good on my side. I don't think `.env` will overwrite exported environment values, but if it does, whoever is responsible for the deployment will need to move the production exported values to `.env`.

Reply:
From the [django-dotenv](https://github.com/jpadilla/django-dotenv) repository:
```
By default, variables that are already defined in the environment take precedence over those in your .env file. To change this, call read_dotenv(override=True).
```

For deploying to the existing environments, this change should have no impact.